reana_client/cli/utils.py

```diff
@@ -136,6 +136,21 @@ def validate_workflow_name(ctx, _, workflow_name):
     return workflow_name
 
 
+def params_tuple_to_dict(ctx, param, value):
```

Comment:
Could we make the name more explanatory? `params_tuple_to_dict` seems too low level, what about more high level, `key_value_to_dict` because it takes`-o KEY=VALUE` and gives `{'KEY': 'VALUE'}` or something simpler.